Riot/Assets/bg.lproj/Vector.strings

```diff
@@ -1145,3 +1153,116 @@
 "secrets_setup_recovery_passphrase_confirm_information" = "Въведете защитната парола отново за да потвърдите.";
 "secrets_setup_recovery_passphrase_confirm_passphrase_title" = "Потвърди";
 "secrets_setup_recovery_passphrase_confirm_passphrase_placeholder" = "Потвърдете паролата";
+
+
+//TODO: Translate these strings
+"store_promotional_text" = "Privacy-preserving chat and collaboration app, on an open network. Decentralised to put you in control. No datamining, no backdoors and no third party access.";
```

Comment:
A lot of (if not all of) the string changes in this build, were a result of me trying to make consistent the way we deal with untranslated strings.

In the matrix repos, untranslated strings are left untranslated, until they're translated. In the app, this means a title that usually reads "Directory" in English, may read as "tab_directory_title" in a language that does not have a translated string.

For Lingo, the method was to put an English version of each new string in each language file, but to make more consistent this treatment of translations in this repository, I ran a diff on the strings files and pulled across missing translations where they appeared. I can of course revert this change if necessary, in which case it would be great to change to writing English strings, and leaving other languages untranslated, as is the case in the base matrix repository.

Reply:
I suggest we ignore all internationalisation for now, I haven't seen internationalisation in the acceptance criteria on the user stories, so I would be surprised if the testers have been testing this.  Instead a story can be created for internationalisation and address it all at once then.